### DISCUSSION TITLE
It's mandatory to create a horizontal and vertical scroll mode for datatable

### DISCUSSION THREAD

**Discussion by jvalle63** [flet-dev/flet#2563] [Q&A] [answered]
**Please Describe The Problem To Be Solved**

(I urgently need the scroll functionality with scroll bar, for datatables.
Currently it is not possible to show more than three columns of a datatable, depending on the size of the cell phone.
I searched the Flutter documentation, and there is this functionality there...

**(Optional): Suggest A Solution**

I think it should work like responsive rolls, where we could indicate the minimum number of columns we want to show, without scrolling
* Caveats and considerations for the future

If there are multiple solutions, please present each one separately. Save comparisons for the very end.)

**Comment by ndonkoHenri** [comment, marked as answer]
Wrap your `Datatable` with `Column` for vertical scroll, and/or `Row` for horizontal scroll.

**Reply by jvalle63** [reply]
The combination of column and row scroll works fine. Thanks a lot
